app/code/community/Bolt/Boltpay/controllers/Adminhtml/Sales/Order/CreateController.php

```diff
@@ -72,28 +72,36 @@ public function loadBlockAction()
         parent::loadBlockAction();
     }
 
+
     /**
      * Saving quote and create order.  We add the Bolt reference to the session
      */
     public function saveAction()
     {
-        // some versions of Magento store the shipping method at the top level of the $_POST array
-        if ($this->getRequest()->getPost('shipping_method')) {
-            $_POST['order']['shipping_method'] = $this->getRequest()->getPost('shipping_method');
-        }
-
-        // We must assure that Magento knows to recalculate the shipping
-        $_POST['collect_shipping_rates'] = 1;
 
         /////////////////////////////////////////////////////////////////////////////
         // If there is no bolt reference, then it indicates this is another payment
         // method.  In this case, we differ to Magento to handle this
         /////////////////////////////////////////////////////////////////////////////
         $boltReference = $this->getRequest()->getPost('bolt_reference');
-
         if (!$boltReference) {
+            $this->_normalizeOrderData();
             parent::saveAction();
             return;
+        } else {
```

Comment:
no need for "else" because you return in if branch above.

Reply:
@daisy1754 Thank you.  I agree.  This was specifically put in for "easier reading" to highlight the intentional nature of the two separate cases because this was called into question in the parent branch.

Had it not been the case, I would have left it as it was in the parent branch where the understanding of data normalization for two cases was to be implicitly understood.  This is also why in the subject of this PR, I have written:
```
Made it even more clear the intentional nature of the normalization code being executed prior to deferring control to Magento for all non-Bolt orders
Explicitly wrote out a call to data normalization code after the immutable quote is stored to the admin session for clarity
```

However, since it's intentional nature, I think at this point, is very clear, it's definitely very ok to remove the else statement.  

**Most importantly, the pull request is now a request to be merged into `develop`, and not the original parent branch.**